numba/unicode.py

```diff
@@ -1056,6 +1056,40 @@ def impl(a):
         return impl
 
 
+@overload_method(types.UnicodeType, 'istitle')
+def unicode_istilte(a):
+    """
+    Implements .istitle()
+    """
+
+    # impl is an approximate translation of:
+    # https://github.com/python/cpython/blob/1d4b6ba19466aba0eb91c4ba01ba509acf18c723/Objects/unicodeobject.c#L11892-L11948
```

Comment:
@esc No it points to unicode_istitle_impl. I had checked it before submitting it.

Reply:
@rdesai16 it also points to the strange place noted by @esc for me, somewhere straddling a comment and a couple of functions, the referenced block starts with `A string is whitespace if all characters in the string are whitespace and there`!? Please could you double check? Thanks.

Reply:
Oh! Okay I'll double check to see if I am looking at the updated codebase.

Reply:
Thanks. It may be that in the current master branch those are the right lines, but that URL has a SHA (and this is deliberate to create a permanent reference) and so the lines needs to be correct with respect to that specific point in the history.

Reply:
CC @shssf